runelite-api/src/main/java/net/runelite/api/ScriptID.java

```diff
@@ -67,6 +67,11 @@
 	 */
 	public static final int CLEAR_CHATBOX_PANEL = 677;
 
+	/**
+	 * Builds the chatbox input widget
+	 */
+	public static final int CHATBOX_INPUT_BUILDER = 223;
```

Comment:
maybe `CHAT_PROMPT_INIT` or something. We have `RUNELITE_CHATBOX_INPUT_INIT` which is similar to this name, but describes something completely different.